src/ssh.rs

```diff
@@ -338,12 +338,16 @@ impl Domain for RemoteSshDomain {
         &self,
         size: PtySize,
         command: Option<CommandBuilder>,
+        command_dir: Option<String>,
```

Comment:
FWIW, the ssh protocol doesn't define a way to specify the cwd, so it would be acceptable to just ignore this parameter here.
In theory you could augment the program that we run to turn it into `cd CWD && CMD`, but we'd need to be confident that the remote system was running a compatible shell, and that isn't guaranteed these days now that Windows can run openssh and spawn `cmd.exe` on it.